updating-to-0-20.md

```diff
@@ -0,0 +1,147 @@
+# Breaking changes
+
+## Our init file, app.js
+
+Starting from your **app.js**, since this version is classbased now, the
+way we configure our bots has been changed, remember this?
+
+```js
+const komada = require("komada");
+
+komada.start({
+  botToken: "your-bot-token",
+  ownerID: "your-user-id",
+  clientID: "the-invite-app-id",
+  prefix: "+",
+  clientOptions: {
+    fetchAllMembers: true
+  }
+});
+```
+
+Now, your bot token **must** go in a separated function: `client.login`. And instead of
+`komada.start`, we create a new instance of `Komada.Client` and assign it to any variable,
+for example `client`, and with this variable, call the function login.
+
+```js
+const Komada = require("komada");
+
+const client = new Komada.Client({
+  ownerID: "your-user-id",
+  clientID: "the-invite-app-id",
+  prefix: "+",
+  clientOptions: {
+    fetchAllMembers: true,
+  },
+});
+
+client.login("your-bot-token");
+```
+
+Additionally, if you want to change the way permission levels are configured, you
+can use a new property: `permStructure`. You can set a permission structure with a
+range of 0-10.
+
+However, it only accepts an array of 11 objects, with the following format:
+
+```js
+permStructure: [
+  {
+    check: () => true,
+    break: false,
+  },
+  {
+    check: () => false,
+    break: false,
+  },
+  {
+    check: (client, msg) => {
+      if (!msg.guild) return false;
+      const modRole = msg.guild.roles.find("name", msg.guild.conf.modRole);
+      return modRole && msg.member.roles.has(modRole.id);
+    },
+    break: false,
+  },
+  {
+    check: (client, msg) => {
+      if (!msg.guild) return false;
+      const adminRole = msg.guild.roles.find("name", msg.guild.conf.adminRole);
+      return adminRole && msg.member.roles.has(adminRole.id);
+    },
+    break: false,
+  },
+  // ...
+],
+```
+
+Which permLevel **0** always return `true` (everyone has this permission level), **1**
+always return `false`, **2** returns either `true` or `false`, depends if the condition
+is fulfilled or not.
+
+You can also use Komada.PermLevels constructor, which is 'easier' to use. In that way, we just do:
+
+```js
+permStructure: new Komada.PermLevels()
+  .addLevel(0, false, () => true)
+  .addLevel(2, false, (client, msg) => {

```

Comment:
Careful using client here, the if you assign new Komada.Client to client, client will already be in the scope...